samples/productSearch/productSets.js

```diff
@@ -160,6 +199,17 @@ require(`yargs`) // eslint-disable-line
     {},
     opts => listProductSets(opts.projectId, opts.location)
   )
+  .command(
```

Comment:
I'm following up with @beccasaurus to learn more about this section in general - I haven't seen it in recent node samples and want to make sure it's still part of our current formatting - but again, I know you're keeping up with the way the rest of the samples are